components/well-bold/well-bold-en.html

```diff
@@ -1,74 +1,50 @@
 ---
-breadcrumbs: [
+breadcrumb: [
```

Comment:
Il y a un "s" pour la propriété breadcrumbs. ref.  https://github.com/wet-boew/GCWeb/blob/master/sites/breadcrumbs/inc-breadcrumbs.html#L12
```suggestion
breadcrumbs: [
```